### PR TITLE
Chapter computational performance/multiple gpus

### PR DESCRIPTION
**Pull request by zhuyuanxiang**

### CODE REVIEW

**Comment by mli** [comment]
Job d2l-zh/PR-840/2 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-840/

File: chapter_computational-performance/multiple-gpus.md [hunk 1]
```diff
@@ -159,7 +159,7 @@ print('b1 weight:', new_params[1])
 print('b1 grad:', new_params[1].grad)
 ```
 
-由于我们还没有进行任何计算，偏置参数的梯度仍然为零。现在假设有一个向量分布在多个GPU上。下面的[**`allreduce`函数将所有向量相加，并将结果广播回所有GPU**]。请注意，要使其工作，我们需要将数据复制到累积结果的设备。
```
Comment (goldmermaid) [inline chapter_computational-performance/multiple-gpus.md:162]: Hey @zhuyuanxiang , could you recover the `[**`, `**]`, `(**`, `**)` symbols back? We are using these symbols for building slides (https://book.d2l.ai/user/slides.html). Thanks!

**Comment by mli** [comment]
Job d2l-zh/PR-840/3 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-840/

File: chapter_computational-performance/multiple-gpus.md (hunk 1 above)
Comment (zhuyuanxiang) [reply chapter_computational-performance/multiple-gpus.md:162]: Hey @goldmermaid, I have recovered the `[**` and `**]` symbols back.

Comment (zhuyuanxiang) [reply chapter_computational-performance/multiple-gpus.md:162]: Thks for your explanation.

**Review by goldmermaid — APPROVED** [review]
Fabulous work @zhuyuanxiang !

**Comment by mli** [comment]
Job d2l-zh/PR-840/4 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-840/

**Pull request merged**